beacon-chain/sync/fork_watcher.go

```diff
@@ -32,10 +32,60 @@ func (s *Service) forkWatcher() {
 						log.WithError(err).Error("Could not retrieve fork digest")
 						continue
 					}
+					digestExists := false
+					for _, t := range s.subHandler.allTopics() {
+						retDigest, err := digestFromTopic(t)
+						if err != nil {
+							log.WithError(err).Error("Could not retrieve digest")
+							continue
+						}
+						if retDigest == digest {
+							digestExists = true
+							break
+						}
+					}
+					if digestExists {
+						continue
+					}
```

Comment:
Can we create a map for this? O(n) lookup is not great.

Reply:
Done, refactored it to be handled in the subscription handler.